config.py

```diff
@@ -48,6 +48,16 @@ class Config(object):
             ReturnCodes.TEMPORARY,
             (True, True),
         ),
+        'compass.mozillafoundation.org': (
+            'https://foundation.mozilla.org/en/404/',
+            ReturnCodes.TEMPORARY,
+            (False, False),
+        ),
+        'movementbuilding.mozillafoundation.org': (
+            'https://foundation.mozilla.org/en/404/',
+            ReturnCodes.TEMPORARY,
+            (False, False),
+        ),

```

Comment:
Thanks @dlopezvsr! Seems like these should be permanent redirects since the websites are decommissioned. I'm not sure if these sites supported different languages, but maybe we should also link to the unlocalized 404 URL and let the website detect language, just in case? Other than that, I think we're good to go!

```suggestion
        'compass.mozillafoundation.org': (
            'https://foundation.mozilla.org/404/',
            ReturnCodes.PERMANENT,
            (False, False),
        ),
        'movementbuilding.mozillafoundation.org': (
            'https://foundation.mozilla.org/404/',
            ReturnCodes.PERMANENT,
            (False, False),
        ),
```

Reply:
Hey @robdivincenzo thanks so much for the observations, you are totally right! I just updated it. Thanks again!